### PR TITLE
Update documentation for injecting anomalies if user not using rke2 cluster

### PR DESCRIPTION
**Pull request by AmartC**
Updated documentation about steps user may need to take if they are not using an RKE2 cluster from the quickstart.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -25,7 +25,18 @@ Make sure to be in the Global Tenant mode if you are not already. Click on Dashb
...
+
+You can also choose to inject different anomalies such as creating pods which are unschedulable, have nonexistent images and exiting with non-zero exit codes. 
+
+Note: If you are not using the quickstart script, make sure to set the KUBECONFIG environment variable like
+```
+export KUBECONFIG=[PATH_TO_KUBECONFIG_FILE]
+```
+and your Kubectl path like
+```
+export PATH=[PATH_TO_KUBECTL_BINARY]
```
Comment (dbason) [inline docs/index.md:37]: This should be
```
export PATH=$PATH:[PATH_TO_KUBECTL_BINARY]
```

**Review by dbason — CHANGES_REQUESTED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -25,7 +25,18 @@ Make sure to be in the Global Tenant mode if you are not already. Click on Dashb
 
 To view the anomaly detection press 'Enter' in the console window running the script to inject an anomaly.
 
-You can also choose to inject different anomalies such as creating pods which are unschedulable, have nonexistent images and exiting with non-zero exit codes.
+
+You can also choose to inject different anomalies such as creating pods which are unschedulable, have nonexistent images and exiting with non-zero exit codes.
```
Comment (kralicky) [inline docs/index.md:29]: Suggest rewording this line to something like
> Using the provided script, you can inject sample anomalies into your cluster. The script can create pods which are unschedulable, have nonexistent images, or exit with non-zero exit codes.

File: docs/index.md [hunk 3]
```diff
@@ -25,7 +25,18 @@ Make sure to be in the Global Tenant mode if you are not already. Click on Dashb
 
 To view the anomaly detection press 'Enter' in the console window running the script to inject an anomaly.
 
-You can also choose to inject different anomalies such as creating pods which are unschedulable, have nonexistent images and exiting with non-zero exit codes.
+
+You can also choose to inject different anomalies such as creating pods which are unschedulable, have nonexistent images and exiting with non-zero exit codes. 
+
+Note: If you are not using the quickstart script, make sure to set the KUBECONFIG environment variable like
```
Comment (kralicky) [inline docs/index.md:31]: Reword "make sure to" -> "you must"

**Review by dbason — APPROVED** [review]

**Review by kralicky — APPROVED** [review]
